web/packages/design/tsconfig.json

```diff
@@ -7,6 +7,12 @@
     "emitDeclarationOnly": true,
     "declarationMap": true,
     "strict": true,
-    "noImplicitReturns": true
+    "noImplicitReturns": true,
+    "paths": {
+      "design/*": ["web/packages/design/src/*"],
```

Comment:
I wanted to shorten the path to `["./src/*"]` but this requires adding `baseUrl: "."` in this `tsconfig.json` (currently we have it set in `tsconfig.base.json`). I decided to leave it as is, especially that it would make the path for`gen-proto-ts/*` longer.